pkg/cache/compressed/cache.go

```diff
@@ -0,0 +1,106 @@
+package compressed
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+
+	// simple checksum usage for validation
+	"crypto/md5" // nolint:gosec
+	"fmt"
+	"time"
+
+	"github.com/openshift/sippy/pkg/apis/cache"
+)
+
+const (
+	cachePrefix = "cc:"
+)
+
+type Cache struct {
+	Cache cache.Cache
+}
+
+func NewCompressedCache(c cache.Cache) (*Cache, error) {
```

Comment:
Doesn't seem to be used at this time? Assume this was an attempt at helping us stick with just redis.

Reply:
Updated to actually use the function and not just init the struct.